doc/minikube_setup.md

```diff
@@ -44,26 +44,26 @@ Create a single node Bookkeeper Cluster using the [BookKeeper Operator](https://
 spec:
   replicas: 1
   storage:
-      ledgerVolumeClaimTemplate:
-        accessModes: [ "ReadWriteOnce" ]
-        storageClassName: "standard"
-        resources:
-          requests:
-            storage: 10Gi
-
-      journalVolumeClaimTemplate:
-        accessModes: [ "ReadWriteOnce" ]
-        storageClassName: "standard"
-        resources:
-          requests:
-            storage: 10Gi
-
-      indexVolumeClaimTemplate:
-        accessModes: [ "ReadWriteOnce" ]
-        storageClassName: "standard"
-        resources:
-          requests:
-            storage: 10Gi
+    ledgerVolumeClaimTemplate:
```

Comment:
I don't see any change in these lines 47 to 66. They have exactly the same content as earlier. Please revert the change.

Reply:
This was an intentional change to correct the indentation of these lines